### PR TITLE
FIX: Remove 3rd button from lightning wallets top

### PR DESCRIPTION
**Pull request by ncoelho**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/hbvz22fec1hm6wbvjja19jfgxw

[download apk](http://filestorage.bluewallet.io:1488/1588169420.apk)

File: screen/wallets/transactions.js [hunk 1]
```diff
@@ -218,7 +218,18 @@ export default class WalletTransactions extends Component {
     return (
       <View style={{ flex: 1 }}>
         <View style={{ flexDirection: 'row', margin: 16, justifyContent: 'space-evenly' }}>
-          {this.state.wallet.getTransactions().length > 0 && this.renderSellFiat()}
+          {/*
```
Comment (Overtorment) [inline screen/wallets/transactions.js:221]: this commens is copypasted and is incorrect

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/1hqfg199xhemjwp078u1ve9pm8

[download apk](http://filestorage.bluewallet.io:1488/1588173010.apk)

**Comment by Overtorment** [comment]
fixed myself

**Pull request merged**